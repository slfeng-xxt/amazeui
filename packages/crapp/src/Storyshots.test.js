import initStoryshots from 'storyshots'
// https://github.com/storybooks/storybook/issues/880
// create .babelrc in .storybook/ folder

initStoryshots()
