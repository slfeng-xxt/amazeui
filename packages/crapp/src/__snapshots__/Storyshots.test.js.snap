exports[`Storyshots Components Name with info 1`] = `
<div>
  <h1>
    Examples
  </h1>
  <div>
    <div
      style={undefined}>
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "color": "#444",
            "fontFamily": "
              -apple-system, \".SFNSText-Regular\", \"San Francisco\", \"Roboto\",
              \"Segoe UI\", \"Helvetica Neue\", \"Lucida Grande\", sans-serif
            ",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
          }
        }>
        <div
          style={
            Object {
              "borderBottom": "1px solid #eee",
              "marginBottom": 10,
              "paddingTop": 10,
            }
          }>
          <h1
            style={
              Object {
                "fontSize": "35px",
                "margin": 0,
                "padding": 0,
              }
            }>
            Components
          </h1>
          <h2
            style={
              Object {
                "fontSize": "22px",
                "fontWeight": 400,
                "margin": "0 0 10px 0",
                "padding": 0,
              }
            }>
            Name with info
          </h2>
        </div>
      </div>
    </div>
    <div>
      <div>
        <div
          className="name ">
          Louie Anderson
        </div>
      </div>
    </div>
    <div
      style={undefined}>
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "color": "#444",
            "fontFamily": "
              -apple-system, \".SFNSText-Regular\", \"San Francisco\", \"Roboto\",
              \"Segoe UI\", \"Helvetica Neue\", \"Lucida Grande\", sans-serif
            ",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
          }
        }>
        <div
          style={
            Object {
              "marginBottom": 0,
            }
          }>
          <p
            style={
              Object {
                "WebkitFontSmoothing": "antialiased",
                "color": "#444",
                "fontFamily": "
                  -apple-system, \".SFNSText-Regular\", \"San Francisco\", \"Roboto\",
                  \"Segoe UI\", \"Helvetica Neue\", \"Lucida Grande\", sans-serif
                ",
                "fontSize": "15px",
              }
            }>
            A component to display a colored name tag.
          </p>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }>
            Story Source
          </h1>
          <pre
            style={
              Object {
                "backgroundColor": "#fafafa",
                "fontFamily": "Menlo, Monaco, \"Courier New\", monospace",
                "fontSize": ".88em",
                "lineHeight": 1.5,
                "overflowX": "scroll",
                "padding": ".5rem",
              }
            }>
            <div
              style={
                Object {
                  "paddingLeft": 18,
                  "paddingRight": 3,
                }
              }>
              <span
                style={
                  Object {
                    "color": "#777",
                  }
                }>
                <
                Name
              </span>
              <span>
                <span>
                   
                  <span
                    style={Object {}}>
                    name
                  </span>
                  <span>
                    =
                    <span
                      style={Object {}}>
                      <span
                        style={
                          Object {
                            "color": "#22a",
                            "wordBreak": "break-word",
                          }
                        }>
                        \"
                        Louie Anderson
                        \"
                      </span>
                    </span>
                  </span>
                   
                </span>
              </span>
              <span
                style={
                  Object {
                    "color": "#777",
                  }
                }>
                />
              </span>
            </div>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "fontSize": "25px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }>
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "margin": "20px 0 0 0",
                }
              }>
              \"
              Name
              \" Component
            </h2>
            <table
              style={
                Object {
                  "borderCollapse": "separate",
                  "borderSpacing": "10px 5px",
                  "marginLeft": -10,
                }
              }>
              <thead>
                <tr>
                  <th>
                    property
                  </th>
                  <th>
                    propType
                  </th>
                  <th>
                    required
                  </th>
                  <th>
                    default
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    type
                  </td>
                  <td>
                    other
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    -
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
